crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -198,15 +198,12 @@ async fn graphiql() -> impl axum::response::IntoResponse {
     )
 }
 
-#[cfg(test)]
```

Comment:
shouldnt we still keep this flag?

Reply:
Tried but but then they wont be visible to the integration tests.

Reply:
We should move this into a test helper crate in that case -- we shouldn't leak test helpers into the default build configuration.